protocols/imap/src/main/java/org/apache/james/imap/api/message/response/StatusResponse.java

```diff
@@ -152,6 +154,13 @@ public static ResponseCode condStore(IdRange[] failedRanges) {
             return new ResponseCode("MODIFIED", Arrays.asList(new String[] { failed}), 0, false);
         }
         
+        /** RFC4551 <code>Conditional STORE</code> response code */
```

Comment:
It is more than a type change.... It looks like you correct conformity to the RFC here...
It is cool but it may be better not to do it in a 2500+ lines modified commit?